File: RQ3/C-REval-Desc/task02_desc.c
bool below_zero(int* operations, int size) {
    int balance = 0;
    bool _result = false;

    for (int i = 0; i < size; i++) {
        balance += operations[i];
        if (balance < 0) {
            _result = true;
        }
    }

    assert(func(_result)); 
    return _result;
}

File: RQ3/Java-REval-Desc/task02_desc.java
import java.util.List;

public class TranslateCode {
    public static boolean belowZero(List<Integer> operations) {
        int balance = 0;
        boolean _result = false;

        for (int op : operations) {
            balance += op;
            if (balance < 0) {
                _result = true;
            }
        }

        assert func(_result);  // POST: func is equivalent to: You're given a list of deposit and withdrawal operations on a bank account that starts with zero balance. Your task is to detect if at any point the balance of account falls below zero, and at that point function should return True. Otherwise it should return False. >>> below_zero([1, 2, 3]) False >>> below_zero([1, 2, -4, 5]) True
    }
}